src/Logging/HTTPOutputHandler.php

```diff
@@ -70,6 +72,38 @@ public function setStatusCode($statusCode)
     }
 
     /**
+     * Set a formatter to use if Director::is_cli() is true
+     */
+    public function setCliFormatter(FormatterInterface $cliFormatter)
```

Comment:
Not specifically mentioned in PSR-1 but these two references suggest that `setCliFormatter()` would be better:

 * https://groups.google.com/forum/#!topic/php-fig/rGopcf4WyyU
 * http://stackoverflow.com/a/15526526